opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEdition.jsx

```diff
@@ -36,36 +47,72 @@ const StixCoreRelationshipEdition = ({
   handleClose,
   handleDelete,
   noStoreUpdate,
+  inGraph,
 }) => {
   const classes = useStyles();
   const queryRef = useQueryLoading(stixCoreRelationshipEditionOverviewQuery, {
     id: stixCoreRelationshipId,
   });
-  return (
-    <Drawer
-      open={open}
-      anchor="right"
-      elevation={1}
-      sx={{ zIndex: 1202 }}
-      classes={{ paper: classes.drawerPaper }}
-      onClose={handleClose}
-    >
-      {queryRef ? (
-        <React.Suspense fallback={<Loader variant={LoaderVariant.inElement} />}>
-          <StixCoreRelationshipEditionOverview
-            queryRef={queryRef}
-            handleClose={handleClose}
-            handleDelete={
-              typeof handleDelete === 'function' ? handleDelete : null
-            }
-            noStoreUpdate={noStoreUpdate}
-          />
-        </React.Suspense>
-      ) : (
-        <Loader variant={LoaderVariant.inElement} />
-      )}
-    </Drawer>
-  );
+  const renderClassic = () => {
+    return (
+      <Drawer
+        open={open}
+        anchor="right"
+        elevation={1}
+        sx={{ zIndex: 1202 }}
+        classes={{ paper: classes.drawerPaper }}
```

Comment:
Only the class changes between renderClassic and renderInGraph, can we only add : 

`classes={{ paper: inGraph ? classes.drawerPaperInGraph : classes.drawerPaper }}`

Instead of rewrite the whole function ?

Suggested changes:

```js
  return (
    <Drawer
      open={open}
      anchor="right"
      elevation={1}
      sx={{ zIndex: 1202 }}
      classes={{ paper: inGraph ? classes.drawerPaperInGraph : classes.drawerPaper }} //-- HERE
      onClose={handleClose}
    >
      {queryRef ? (
        <React.Suspense fallback={<Loader variant={LoaderVariant.inElement} />}>
          <StixCoreRelationshipEditionOverview
            queryRef={queryRef}
            handleClose={handleClose}
            handleDelete={
              typeof handleDelete === 'function' ? handleDelete : null
            }
            noStoreUpdate={noStoreUpdate}
          />
        </React.Suspense>
      ) : (
        <Loader variant={LoaderVariant.inElement} />
      )}
    </Drawer>
  );
```